controllers/saml2aws_controller.go

```diff
@@ -121,6 +110,13 @@ func (r *Saml2AwsReconciler) Reconcile(req ctrl.Request) (ctrl.Result, error) {
 	return ctrl.Result{RequeueAfter: time.Minute * time.Duration(1)}, nil
 }
 
+func (r *Saml2AwsReconciler) updateStatus(ctx context.Context, saml *samletv1.Saml2Aws, err error) error {
+	if err != nil {
+		saml.Status.State = err.Error()
```

Comment:
My understanding is that according to https://github.com/AbsaOSS/samlet/issues/32, `spec.Status.State` should be always updated, not only in case of Error, and should be one of `Provisioning|Error|Completed`
And there should be an additional `spec.Status.Error` field to contain the error message in case of error.

Reply:
Right it was lost during stash.